City-of-Pori/pori-events
# Feature/pori 230 events2elastic

Please merge this first: https://github.com/City-of-Pori/pori-events/pull/1

How to test:
- `composer update --with-dependencies` 
- Ensure `events2elastic` module is on
- `drush cim`
- Go create/edit one event node
- You should see JSON output in https://local.tapahtumat.pori.fi/api/search/searchkit/event-node/_search
